llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -750,7 +750,7 @@ END_TWO_BYTE_PACK()
   bool use_empty() const { return UseList == nullptr; }
 
   /// Return true if there is exactly one use of this node.
-  bool hasOneUse() const { return hasSingleElement(uses()); }
+  bool hasOneUse() const { return hasSingleElement(users()); }
```

Comment:
Changing this name will be difficult, but maybe it is worth fixing the comment at least?
This method returns one if all users are the same node.

Reply:
It looks like I'm wrong. I had a strong feeling that this returns true if all uses are the same node, but after looking around it does not appear so. Probably mistaken for some other method. Sorry.

Reply:
>This method returns true if all users are the same node.

That's not true. hasSingleElement just checks that the range contains 1 element. We don't do anything to remove duplicate users from the range.

Reply:
Yep, sorry.